projects/owa-data/scripts/conversion/CS_DM/convert_to_owamcap.py

```diff
@@ -1,4 +1,10 @@
 #!/usr/bin/env python3
+# /// script
+# requires-python = ">=3.11"
```

Comment:
I guess much more dependencies are needed here? e.g. owa-core, mcap-owa-support, ...
Did you tested this with `uv run abcd.py`?

Reply:
Oh I thought the script dependencies were incremental, but that doesn't seem to be the case.

Reply:
done!